smarts/core/scenario.py

```diff
@@ -66,6 +66,11 @@
 
 VehicleWindow = TrafficHistory.TrafficHistoryVehicleWindow
 
+# Suppress trimesh deprecation warning
+
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    import trimesh # only suppress the warnings caused by trimesh
```

Comment:
+ Does the catch warning also cater to `import trimesh` in other files, such as `opendrive_road_network.py`, `sumo_road_network.py`, `waymo_map.py`, and `geometry.py`? 
+ Can catch warning be implemented inside `map_glb_meta_for_file()` method below in this file where `trimesh` is being imported?

Reply:
@qianyi-sun Could you make the warning more specific? It is possible to pattern match warnings to ignore.

Reply:
@Adaickalavan I checked running waymo/opendrive/sumo scenarios, seems like the warning is gone in all three type of scenarios. 
@Gamenot If I understood "specfic" correctly, I just added a `message=` option in the `warnings.filterwarning` to only suppress this specific message.

Reply:
Yes, that is what I meant.